types/fingerprintjs2/index.d.ts

```diff
@@ -1,46 +1,87 @@
-// Type definitions for fingerprintjs2 1.5
+// Type definitions for fingerprintjs2 2.0
```

Comment:
When upgrading the major version, please follow the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version